pkg/controller/hostpathprovisioner/contoller_test.go

```diff
@@ -120,15 +124,15 @@ var _ = Describe("Controller reconcile loop", func() {
 			},
 		}
 		saNN := types.NamespacedName{
-			Name:      "test-name-admin",
+			Name:      "hostpath-provisioner-admin",
 			Namespace: "test-namespace",
 		}
 		cr, r, cl = createDeployedCr(cr)
 		// Now modify the service account to something not desired.
 		sa := &corev1.ServiceAccount{}
 		err := cl.Get(context.TODO(), saNN, sa)
 		Expect(err).NotTo(HaveOccurred())
-		Expect(sa.ObjectMeta.Labels["k8s-app"]).To(Equal("test-name"))
+		Expect(sa.ObjectMeta.Labels["k8s-app"]).To(Equal("hostpath-provisioner"))
```

Comment:
Use the const you just create in common.go?

Reply:
Done